ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/commands/CommandCompletionServiceIT.scala

```diff
@@ -154,6 +162,64 @@ class CommandCompletionServiceIT
             commandIds3 should not contain (commandIds1(1))
           }
       }
+
+      "implicitly tail the stream if no offset is passed" in allFixtures { ctx =>
+        // A command to create a Test.Dummy contract (see //ledger/sandbox/src/main/resources/damls/Test.daml)
+        val createDummyCommand = ProtoCommand(
+          ProtoCommand.Command.Create(
+            ProtoCreateCommand(
+              templateId = Some(templateIds.dummy),
+              createArguments = Some(
+                Record(
+                  fields = Seq(
+                    RecordField("operator", Some(Value(Value.Sum.Party(party))))
+                  )
+                ))
+            )))
+
+        // Issue the `createDummyCommand` to the ledger using the managed command client
+        def createDummy(id: String) =
+          for {
+            commandClient <- ctx.commandClient(ctx.ledgerId)
```

Comment:
you don't have to create a client for every command you send, it's probably not too efficient

Reply:
Good catch, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1961/commits/1f8fae9ab62c717e1424427319070ee4e186564f